### PR TITLE
SCA

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

File: src/Console/Command/DocumentationCommand.php [hunk 1]
```diff
@@ -68,6 +69,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return 0;
     }
 
+    /**
+     * @param FixerInterface[] $fixers
```
Comment (GrahamCampbell) [inline src/Console/Command/DocumentationCommand.php:73]: `list<FixerInterface>` would be better. `FixerInterface[]` is an alias for `array<int|string, FixerInterface>`

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Comment by julienfalque** [comment]
Are those changes reported by PHPStan or another tool?

**Comment by liquid207** [comment]
PHPStan is really suitable for doing this.

**Comment by SpacePossum** [comment]
stan, psalm, phpstorm build in, https://plugins.jetbrains.com/plugin/10215-php-inspections-ea-ultimate- and some custom "fixers"

File: src/Console/Command/DocumentationCommand.php (hunk 1 above)
Comment (SpacePossum) [reply src/Console/Command/DocumentationCommand.php:73]: Would list suggest that the type of index is not defined or not of interest, does it also implies the order of the elements is trivial? Would be nice to have some documentation backing this up as I think it is not in PSR5?

**Comment by SpacePossum** [comment]
closing for now, I'll make a new one when we are aligned on the hash/list/array syntax, thanks for reviewing all :+1:

File: src/Console/Command/DocumentationCommand.php (hunk 1 above)
Comment (julienfalque) [reply src/Console/Command/DocumentationCommand.php:73]: `list<T>` means `array<int, T>` where keys start at 0 and are consecutive without gaps.

**Pull request closed without merging**